src/content/developers/docs/nodes-and-clients/run-a-node/index.md

```diff
@@ -460,6 +460,7 @@ As part of your monitoring, make sure to keep an eye on your machine's performan
 ## Further reading {#further-reading}
 
 - [Guide | How to setup a validator for Ethereum staking on mainnet](https://www.coincashew.com/coins/overview-eth/guide-or-how-to-setup-a-validator-on-eth2-mainnet) _– CoinCashew, updated regularly_
+- [Ethereum Staking Guides](https://github.com/SomerEsat/ethereum-staking-guides) _– Somer Esat, updated regularly_
 - [ETHStaker guides on running validators on testnets](https://github.com/remyroy/ethstaker#guides) – _ETHStaker, updated regularly_
```

Comment:
[As per our framework for product ordering on our listing criteria](https://ethereum.org/en/contributing/adding-products/#product-ordering), this should go at the bottom of the 'staking guides'.

```suggestion
- [ETHStaker guides on running validators on testnets](https://github.com/remyroy/ethstaker#guides) – _ETHStaker, updated regularly_
- [Ethereum Staking Guides](https://github.com/SomerEsat/ethereum-staking-guides) _– Somer Esat, updated regularly_
```

Reply:
Meta comment: this page is about running a node, so the node running further reading should probably be placed above the staking further reading (should we even have staking information here vs pointing people to /staking/?)

cc: @jmcook1186 @wackerow any thoughts?

Reply:
```suggestion
- [ETHStaker guides on running validators on testnets](https://github.com/remyroy/ethstaker#guides) – _ETHStaker, updated regularly - non-validating node operators can simply omit sections on staking and starting validators_
- [Ethereum Staking Guides](https://github.com/SomerEsat/ethereum-staking-guides) _– Somer Esat, updated regularly - non-validating node operators can simply omit sections on staking and starting validators_
```